src/cmd-buildextend-metal

```diff
@@ -122,7 +122,8 @@ fi
 path=${PWD}/${img}
 
 echo "Estimating disk size..."
-/usr/lib/coreos-assembler/estimate-commit-disk-size --repo "$ostree_repo" "$ref" --add-percent 20 > "$PWD/tmp/ostree-size.json"
+# Keep this number 512 in sync with create_disk.sh
+/usr/lib/coreos-assembler/estimate-commit-disk-size --add-mb 512 --repo "$ostree_repo" "$ref" > "$PWD/tmp/ostree-size.json"
 size="$(jq '."estimate-mb".final' "$PWD/tmp/ostree-size.json")"
 # extra size is the non-ostree partitions, see create_disk.sh
 size="$(( size + 513 ))M"
```

Comment:
Then we can drop this, right?

Reply:
Ugh, I somehow missed that.  That throws into question my whole assumption about why I was running out of space.